test/timearray.jl

```diff
@@ -80,13 +80,30 @@ facts("conversion methods") do
     end
 end
 
+facts("index by integer works with both 1d and 2d time array") do
+
+    context("1d time array") do
+      @fact cl[1].timestamp --> [Date(2000,1,3)]
+      @fact cl[1].values    --> [111.94]
+      @fact cl[1].colnames  --> ["Close"]
+      @fact cl[1].meta      --> "AAPL"
+    end
+
+    context("2d time array") do
+      @fact ohlc[1].timestamp --> [Date(2000,1,3)]
+      @fact ohlc[1].values    --> [104.88 112.5 101.69 111.94]
+      @fact ohlc[1].colnames  --> ["Open", "High", "Low","Close"]
+      @fact ohlc[1].meta      --> "AAPL"
```

Comment:
Sorry, yes, I meant the `@fact` lines, which are indented with 2 spaces rather than 4.